12-say-grpc/backend/vendor/github.com/campoy/justforfunc/13-cgo-flite/flite/flite.go

```diff
@@ -0,0 +1,55 @@
+package flite
+
+// #cgo CFLAGS: -I /usr/include/flite
+// #include <flite.h>
+// #include <stdlib.h>
+// #cgo LDFLAGS: -L /usr/lib/x86_64-linux-gnu -lflite -lflite_cmu_us_kal -lflite_usenglish
+// extern cst_voice *register_cmu_us_kal(const char *voxdir);
+import "C"
+import (
+	"fmt"
+	"io/ioutil"
+	"unsafe"
+)
+
+var voice *C.cst_voice
+
+func init() {
+	C.flite_init()
```

Comment:
Don't you also need to destroy this?

I'd do something like

f, err := flite.Open()
f.Close() and destroy voice in Close.